prime-angular-frontend/src/app/modules/enrolment/pages/obo-sites-page/obo-sites-page.component.html

```diff
@@ -32,61 +32,29 @@
-                              [formGroupName]="i">
-
-                  <app-obo-site-form [form]="site"
-                                     [index]="i"
-                                     [careSettingCode]="careSetting.careSettingCode"
-                                     [site]="site"
-                                     [total]="communityHealthSites.controls.length"
-                                     [last]="last"
-                                     [jobNames]="jobNames"
-                                     [allowDefaultOption]="allowDefaultOption"
-                                     [defaultOptionLabel]="defaultOptionLabel"
-                                     (remove)="removeOboSite($event, careSetting.careSettingCode)"></app-obo-site-form>
+                <ng-container
+                              *ngTemplateOutlet="careSettingTemplate; context:{ $implicit: careSetting, careSettingSites: communityHealthSites };">
+                </ng-container>
+              </ng-container>
 
+              <ng-container *ngSwitchCase="CareSettingEnum.DEVICE_PROVIDER"
+                            formArrayName="deviceProviderSites">
+
+                <ng-container
+                              *ngTemplateOutlet="careSettingTemplate; context:{ $implicit: careSetting, careSettingSites: deviceProviderSites };">
                 </ng-container>
-                <button mat-button
-                        class="mb-4"
-                        type="button"
-                        color="primary"
-                        (click)="addOboSite(careSetting.careSettingCode)">
-                  <mat-icon>add</mat-icon>
-                  Add Additional Site
-                </button>
+
               </ng-container>
 
               <ng-container *ngSwitchCase="CareSettingEnum.COMMUNITY_PHARMACIST"
                             formArrayName="communityPharmacySites">
 
-                <ng-container *ngFor="let site of communityPharmacySites.controls; let i = index; let last = last"
-                              [formGroupName]="i">
-
-                  <app-obo-site-form [form]="site"
-                                     [index]="i"
-                                     [careSettingCode]="careSetting.careSettingCode"
-                                     [site]="site"
-                                     [total]="communityPharmacySites.controls.length"
-                                     [last]="last"
-                                     [jobNames]="jobNames"
-                                     [allowDefaultOption]="allowDefaultOption"
-                                     [defaultOptionLabel]="defaultOptionLabel"
-                                     (remove)="removeOboSite($event, careSetting.careSettingCode)"></app-obo-site-form>
-
+                <ng-container
+                              *ngTemplateOutlet="careSettingTemplate; context:{ $implicit: careSetting, careSettingSites: communityPharmacySites };">
```

Comment:
To reduce line length you can use ngTemplateOutletContext to pass context, and use new lines to space out the passed object literal

Reply:
split into [ngTemplateOutlet] and [ngTemplateOutletContext]